concordium-contracts-common-derive/src/lib.rs

```diff
@@ -163,6 +214,16 @@ pub fn deserial_with_state_derive(input: TokenStream) -> TokenStream {
 /// If the type is an enum then all fields of each of the variants must
 /// implement the [`SchemaType`] trait.
 ///
+/// ## Specifying the tag value for an enum variant
+///
+/// When deriving `Serial`, `Deserial` and `DeserialWithState` the
+/// discriminating tag can be set explicitly using `#[concordium(tag = n)]`
+/// where `n` is a unsigned integer literal and annotating the enum with
```

Comment:
"and annotating the enum"?

Something seems off.

Reply:
I have rephrased it a bit